app/javascript/retrospring/features/settings/theme.ts

```diff
@@ -67,7 +67,7 @@ const getDecimalTripletsFromHex = (hex: string): string => {
 }
 
 export function themeDocumentHandler(): void {
-  if (!document.querySelector('#update_theme')) return;
+  if (!document.querySelector('[action="/settings/theme"]')) return;
```

Comment:
Still need to update this, the new route for editing the theme is `/settings/theme/edit`.